snapcraft/meta.py

```diff
@@ -123,10 +122,11 @@ def _compose_snap_yaml(meta_dir, config_data):
             snap_yaml[key_name] = config_data[key_name]
 
     if 'apps' in config_data:
-        apps = config_data['apps']
-        apps = _wrap_apps(apps)
-        snap_yaml['apps'] = \
-            _copy_security_profiles(meta_dir, apps)
+        snap_yaml['apps'] = _wrap_apps(config_data['apps'])
+
+    if 'uses' in config_data:
+        snap_yaml['uses'] = \
+            _copy_security_profiles(meta_dir, config_data['uses'])
```

Comment:
There are no `offers` to copy. The `migration-skill` only applies to `uses`. In theory it could for frameworks, but we removed support for them in snapcraft.

This code will go away as soon as skill `type`s start landing into the `os` and the `migration-skill` is gone.